### PR TITLE
PT-3918: Update CsvHelper nuget package to latest versions

### PR DESCRIPTION
**Pull request by BelyakovSergey**
## Description
Update CsvHelper nuget package to latest versions
### Jira-link:
https://virtocommerce.atlassian.net/browse/PT-3918

### CODE REVIEW

**Review by akak1977 — APPROVED** [review]

**Review by ksavosteev — APPROVED** [review]

File: src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogExporter.cs [hunk 1]
```diff
@@ -68,11 +70,16 @@ public async Task DoExportAsync(Stream outStream, CsvExportInfo exportInfo, Acti
             var criteria = ProductSearchCriteriaFactory(exportInfo);
 
             // Second time: fetch and save to csv
-            var streamWriter = new StreamWriter(outStream, Encoding.UTF8, 1024, true) { AutoFlush = true };
-            using (var csvWriter = new CsvWriter(streamWriter))
+            var streamWriter = new StreamWriter(outStream, Encoding.UTF8, 1024, true) { AutoFlush = true };
+
+            var writerConfig = new CsvConfiguration(CultureInfo.InvariantCulture)
```
Comment (Vectorfield4) [inline src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogExporter.cs:75]: Move to the protected virtual method.

File: src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs [hunk 2]
```diff
@@ -111,18 +112,19 @@ public Task DoImportAsync(Stream inputStream, CsvImportInfo importInfo, Action<E
             };
             progressCallback(progressInfo);
 
-            var encoding = DetectEncoding(inputStream);
-
-            using (var reader = new CsvReader(new StreamReader(inputStream, encoding)))
+            var encoding = DetectEncoding(inputStream);
+
+            var readerConfig = new CsvConfiguration(CultureInfo.InvariantCulture)
```
Comment (Vectorfield4) [inline src/VirtoCommerce.CatalogCsvImportModule.Data/Services/CsvCatalogImporter.cs:117]: Move to the protected virtual method.

File: src/VirtoCommerce.CatalogCsvImportModule.Web/Controllers/Api/CatalogModuleExportImportController.cs [hunk 3]
```diff
@@ -141,17 +143,18 @@ public async Task<ActionResult<CsvProductMappingConfiguration>> GetMappingConfig
...
             var decodedDelimiter = HttpUtility.UrlDecode(delimiter);
-            result.Delimiter = decodedDelimiter;
-
-            //Read csv headers and try to auto map fields by name
-
-            using (var reader = new CsvReader(new StreamReader(_blobStorageProvider.OpenRead(fileUrl))))
+            result.Delimiter = decodedDelimiter;
+
+            //Read csv headers and try to auto map fields by name
+            var config = new CsvConfiguration(CultureInfo.InvariantCulture)
```
Comment (Vectorfield4) [inline src/VirtoCommerce.CatalogCsvImportModule.Web/Controllers/Api/CatalogModuleExportImportController.cs:149]: Move to the protected virtual method.

File: tests/VirtoCommerce.CatalogCsvImportModule.Test/MappingTests.cs [hunk 4]
```diff
@@ -224,12 +225,15 @@ public void CsvHeadersExportTest_DefaultConfiguration_HeadersAreSame()
         {
             using (var sw = new StringWriter())
             {
-                using (var csv = new CsvWriter(sw))
+                var exportInfo = new CsvExportInfo { Configuration = CsvProductMappingConfiguration.GetDefaultConfiguration() };
+                var writerConfig = new CsvConfiguration(CultureInfo.InvariantCulture)
```
Comment (Vectorfield4) [inline tests/VirtoCommerce.CatalogCsvImportModule.Test/MappingTests.cs:229]: Move to the protected virtual method.

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**